src/pybind/mgr/cephadm/agent.py

```diff
@@ -329,6 +330,13 @@ def run(self) -> None:
                 agent_response = secure_agent_socket.recv(1024).decode()
                 self.mgr.log.debug(f'Received "{agent_response}" from agent on host {self.host}')
                 self.mgr.cache.sending_agent_message[self.host] = False
+                if self.daemon_spec:
+                    # agent successfully received new config. Update config/deps
+                    self.mgr.cache.update_daemon_config_deps(
+                        self.daemon_spec.host, self.daemon_spec.name(), self.daemon_spec.deps, datetime_now())
+                    self.mgr.cache.agent_timestamp[self.daemon_spec.host] = datetime_now()
+                    self.mgr.cache.agent_counter[self.daemon_spec.host] = 1
+                    self.mgr.cache.save_host(self.daemon_spec.host)
```

Comment:
```suggestion
                self.mgr.cache.got_agent_response(host, daemon_spec)
```
Let's avoid 5 consecutive lines starting with `self.mgr.cache....` 
```